cosine_similarity_model.py

```diff
@@ -0,0 +1,179 @@
+import json
+import numpy as np
+from sklearn.metrics.pairwise import cosine_similarity
+from pprint import pprint
+from collections import Counter
+import logging
+
+def setup_logger():
+    logger = logging.getLogger('logfile.log')
+    logger.setLevel(logging.INFO)
+    handler = logging.StreamHandler()
+    formatter = logging.Formatter('%(asctime)s - %(levelname)s - %(message)s')
+    handler.setFormatter(formatter)
+    logger.addHandler(handler)
+    return logger
+
+
+logger = setup_logger()
+
+
+def load_movies_from_json(file_path):
+    with open(file_path, "r", encoding="utf-8") as file:
```

Comment:
load movies from the mongo db
you need to use the dataset available in Valeriya
start the crawler application, which will create the necessary collections automatically during startup